### PR TITLE
Add CTR Lp Tvl

### PR DESCRIPTION
**Pull request by jianfengxiaojianfeng**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/concentrator exports TVL: 
        
 
 
    ethereum                  72.07 M
    
    total                    72.07 M

**Comment by llamatester** [comment]
The adapter at projects/concentrator exports TVL: 
        
 
 
    ethereum                  71.76 M
    
    total                    71.76 M

File: projects/concentrator/index.js [hunk 1]
```diff
@@ -30,8 +31,16 @@ const replacements = [
   "0x73a052500105205d34Daf004eAb301916DA8190f"
 ].map(i => i.toLowerCase())
 
+async function getBalancerLpTvl(balances) {
+  const res = await fetchURL('https://concentrator-api.aladdin.club/data/aladdin/initInfo')
```
Comment (vrtnd) [inline projects/concentrator/index.js:35]: Please, can u use rpc calls instead of api?

Comment (jianfengxiaojianfeng) [reply projects/concentrator/index.js:35]: Because CTR price not support on coingecko.com

Comment (vrtnd) [reply projects/concentrator/index.js:35]: U can use defillama pricing api to get price of CTR
https://defillama.com/docs/api

**Comment by llamatester** [comment]
The adapter at projects/concentrator exports TVL: 
        
 
 
    ethereum                  73.57 M
    
    total                    73.57 M

**Pull request merged**